kms_log/main.tf

```diff
@@ -558,4 +593,469 @@ resource "aws_cloudwatch_metric_alarm" "dead_letter" {
     alarm_actions = [
         "${var.sns_topic_dead_letter_arn}"
     ]
+}
+
+#lambda functions
+resource "aws_lambda_function" "cloudtrail_processor" {
+    count = "${var.kmslogging_service_enabled}"
+    s3_bucket = "${data.aws_s3_bucket.lambda.id}"
+    s3_key = "circleci/identity-lambda-functions/${var.lambda_identity_lambda_functions_gitrev}.zip"
+
+    lifecycle {
+        ignore_changes = ["s3_key", "last_modified"]
+    }
+
+    function_name = "${var.env_name}-kmslog-ct-processor"
+    description = "18F/identity-lambda-functions: KMS CT Log Processor"
+    role = "${aws_iam_role.cloudtrail_processor.arn}"
+    handler = "main.Functions::KMSCloudTrailHandler.process"  #TODO Verify
+    runtime = "ruby2.5"
+    timeout = 30 # seconds
+
+    environment {
+        variables = {
+            DEBUG = "${var.lambda_audit_github_debug ? "1" : ""}"
+            LOG_LEVEL = "0"
+            CT_SQS_QUEUE = "${aws_sqs_queue.kms_ct_events.id}"
```

Comment:
Is this a URL? It looks like we need a URL over here: https://github.com/18F/identity-lambda-functions/compare/crissupb/kmslogct#diff-671f94bef1a85fb247d323a0c7f85f45R10

Reply:
Yes it is the URL